src/components/atoms/Radio/index.tsx

```diff
@@ -1,13 +1,20 @@
-import {ReactNode} from 'react'
+import { ReactNode } from 'react'
+import { useAppSelector } from 'store/hooks'
+import cn from 'classnames'
 import './style.scss'
 type RadioProps = {
   children: ReactNode
+  id: string
```

Comment:
Id здесь не нужен, так как наш `<input>` обернут в `<label>`.

Если `<input>` находится вне `<label>`, в таком случае добавляется атрибут `htmlFor` для элемента `<label>`.

```suggestion
  id: string
```
```suggestion
```